pkg/controllers/disruption/expiration_test.go

```diff
@@ -72,476 +75,585 @@ var _ = Describe("Expiration", func() {
 		})
 		nodeClaim.StatusConditions().MarkTrue(v1beta1.Expired)
 	})
-	It("should ignore nodes without the expired status condition", func() {
-		_ = nodeClaim.StatusConditions().ClearCondition(v1beta1.Expired)
-		ExpectApplied(ctx, env.Client, nodeClaim, node, nodePool)
-
-		// inform cluster state about nodes and nodeclaims
-		ExpectMakeNodesAndNodeClaimsInitializedAndStateUpdated(ctx, env.Client, nodeStateController, nodeClaimStateController, []*v1.Node{node}, []*v1beta1.NodeClaim{nodeClaim})
+	Context("Budgets", func() {
```

Comment:
Same question here as with drift: do we need test coverage for cases where we check that we can disrupt with budgets when the pods are bound?